config/externalname.go

```diff
@@ -801,8 +801,8 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	// "azurerm_stream_analytics_function_javascript_udf": config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.StreamAnalytics/streamingjobs/{{ .parameters.job_name }}/outputs/{{ .external_name }}"),
 	// "azurerm_stream_analytics_output_cosmosdb":          config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.StreamAnalytics/streamingjobs/{{ .parameters.stream_analytics_job_id }}/outputs/{{ .external_name }}"),
 	// Does not work, tracked in https://github.com/upbound/official-providers/issues/368
-	// "azurerm_stream_analytics_output_powerbi": config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.StreamAnalytics/streamingjobs/{{ .parameters.stream_analytics_job_id }}/outputs/{{ .external_name }}"),
-	"azurerm_stream_analytics_output_blob": config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.StreamAnalytics/streamingJobs/{{ .parameters.stream_analytics_job_id }}/outputs/{{ .external_name }}"),
+	"azurerm_stream_analytics_output_powerbi": config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.StreamAnalytics/streamingJobs/{{ .parameters.stream_analytics_job_name }}/outputs/{{ .external_name }}"),
```

Comment:
```suggestion
	"azurerm_stream_analytics_output_powerbi": config.TemplatedStringAsIdentifier("name", "/subscriptions/{{ .setup.configuration.subscription_id }}/resourceGroups/{{ .parameters.resource_group_name }}/providers/Microsoft.StreamAnalytics/streamingJobs/{{ .parameters.stream_analytics_job_id }}/outputs/{{ .external_name }}"),
```

Why did we use `.parameters.stream_analytics_job_name` instead of `.parameters.stream_analytics_job_id`. If not, can you test it using the above configuration?

Reply:
The configuration can stay that way, @svscheg. The reason I asked this was that the [latest version of the terraform provider](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/stream_analytics_output_powerbi#example-usage) uses `stream_analytics_job_id` instead of the `stream_analytics_job_name` parameter. But the [terraform version 3.57.0](https://registry.terraform.io/providers/hashicorp/azurerm/3.57.0/docs/resources/stream_analytics_output_table#example-usage) uses `stream_analytics_job_name`(which we use).

We should pay attention to the fix for this in the next provider bump.